New cape (top board) for the Beaglebone Black

cduck opened the pull request:
Just provides power to the BBB and optionally a USB hub through an anderson connector so far.

krystynneisess commented:
POWER-WIRE
- Add device description
- Add tKeepout and bKeepout around hole/vias
- Add tDocu for vias

krystynneisess commented:
**cape.sch:**
- Make all fonts vector (as compared to proportional) fonts.  

**cape.brd--DRC Errors** (Some of these I think you already noticed when looking at the board earlier today, but I'll list them just in case): 
- Angle:  trace connecting vreg's out pad to C2
- Clearance, dimension, and overlap DRC errors for Anderson connectors--caused by the fact that Andersons have copper traces coming from the vias.  (I'm not sure why they need this trace...?)  At any rate, not sure if these'll pass DFM.
     - Clearance:  the trace entering the via is wide enough that it it gets too close to the Anderson trace. Perhaps make power trace thinner?
     - Dimension:  the Anderson is too close to the edge of the board.
     - Overlap:  Anderson traces too close to via.  Cant do too much without changing part, but seems to have been fine in the past.
- Overlap:  A trace on the 12V net is spanning both pads of C1.
-Stop Mask:  
     -  Move Anderson NAME (J1) so that it is on the not-cutout portion of the PCB.

**cape.brd--General**
- Increase dimension layer lines from width 0mil to 10 mil.
- It is unclear what the "U$2" label near the top left corner is for.  It is also not on the board.
- The "BEAGLE_BONE_BLACK_CAPE" label along the bottom edge is not on the board.  (I'm not sure if this was intended?)  
- Add the version, revision, and last modified date in the top copper layer.
- Change all fonts to vector fonts (namely, the board title info, copyright info, and 5V/12V labels).
- Change all font ratios to >= 15%.

cduck commented:
All done.

The Anderson trace is for mechanical support.  The DRC errors due to this is shown as approved on the controller board.  I also modified the Anderson horizontal part so that it knows that the rectangular pad is the same net as the hole.  This fixed the other DRC errors.

Fixed all DRC errors except two for the Anderson connectors.  The same errors were approved on the controller board and didn't cause any issues.

All the extra labels off the board came with the SparkFun Beaglebone part.  I removed them.
